src/base-traits.jl

```diff
@@ -15,12 +15,16 @@ export IsLeafType, IsConcrete, IsBits, IsImmutable, IsContiguous, IsIndexLinear,
 
 "Trait of all isbits-types"
 @traitdef IsBits{X}
-Base.@pure _isbits(X) = X.isbitstype
+Base.@pure _isbits(X) = isbitstype(X)
 @traitimpl IsBits{X} <- _isbits(X)
 
 "Trait of all immutable types"
 @traitdef IsImmutable{X}
-Base.@pure _isimmutable(X) = !X.mutable
+if VERSION >= v"1.7.0-DEV.1279"
+    Base.@pure _isimmutable(x) = !(x.name.flags & 0x2 == 0x2)
```

Comment:
I don't follow.  What is this about?  Tnx!

Reply:
https://github.com/JuliaLang/julia/pull/41018

But actually, I think just using `ismutabletype` works.

Reply:
YOLO github edit...